### PR TITLE
Ide 4466

### PR DESCRIPTION
**Pull request by jtydhr88**

### CODE REVIEW

**Comment by yanan** [comment]
internet problem: ft retest

File: tools/plugins/com.liferay.ide.hook.ui/OSGI-INF/l10n/bundle.properties [hunk 1]
```diff
@@ -3,5 +3,5 @@ bundle.name = Liferay IDE Hook UI
 bundle.vendor = Liferay, Inc.
 category.name = Liferay
 hook.editor.name = Liferay Hook Editor
-hook.wizard.name = Liferay Hook Configuration
+hook.wizard.name = Liferay Hook Configuration (Deprecated)
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.hook.ui/OSGI-INF/l10n/bundle.properties]: hey @jtydhr88  I'm worried about the word 'deprecated'.  I'm not sure we want to list that in the UI.  Lets discuss this on a call next week.

**Comment by gamerson** [comment]
hey @yanan   Can you rename everything here from `Deprecated` to `Legacy` and also rebase and resend to me?  Thanks

**Comment by yanan** [comment]
Got it, will do.

**Pull request closed without merging**